media/js/firefox/new/experiment-search-retention-funnelcakes.js

```diff
@@ -0,0 +1,99 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var experimentId = 'experiment_search_retention_funnelcake';
+    var geoNonmatchCookie = experimentId + '_nonUS';
+
+    // IE 9 can't handle the truth
+    var isIELT10 = /MSIE\s[1-9]\./.test(navigator.userAgent);
+
+    // swiped from mozilla-client.js
+    var ua = navigator.userAgent;
+    var isLikeFirefox = /Iceweasel|IceCat|SeaMonkey|Camino|like\ Firefox/i.test(ua);
+    var isFirefox = /\s(Firefox|FxiOS)/.test(ua) && !isLikeFirefox(ua);
```

Comment:
This is currently throwing an error `TypeError: isLikeFirefox is not a function`